src/internal/puck/components/ThemeHeader.tsx

```diff
@@ -64,29 +64,43 @@ export const ThemeHeader = (props: ThemeHeaderProps) => {
       <div className="header-center"></div>
       <div className="actions">
         <ClearLocalChangesButton
-          disabled={themeHistory?.history.length === 1}
+          disabled={themeHistory?.histories?.length === 1}
           onClearLocalChanges={() => {
             clearThemeHistory();
             if (themeConfig) {
-              updateThemeInEditor(themeHistory?.history[0], themeConfig);
+              updateThemeInEditor(
+                themeHistory?.histories?.[0]?.state
+                  ?.data as unknown as SavedTheme,
```

Comment:
updateThemeInEditor should use the same type as our theme state. We shouldn't need to cast like this if we do things right.